stor/tests/test_dx.py

```diff
@@ -461,3 +462,26 @@ def test_invalid_glob_pattern(self, mock_list):
             dx_p.glob('invalid_*pattern', condition=None)
 
     # TODO(akumar) add tests for retries
+
```

Comment:
Yes, will definitely remove more tests above. This was when I was writing out tests for everything. Since we are going module by module now, will remove tests for glob, listdir and walkfiles too (and reintroduce them later).